### ISSUE TITLE
Add shell.nix for Nix Env Selector 🙏🏻

### ISSUE THREAD

**Issue by kutyel** [rpearce/hakyll-nix-template#43]
Hi, thanks for this template! I'm using vscode with the [Nix Env Selector](https://marketplace.visualstudio.com/items?itemName=arrterian.nix-env-selector) and it does not seem to pick up correctly HLS by default, I guess if we had some shell.nix to point the extension to that had all the libs that we already built in flake.nix we could make the integration with this editor quite seamless, but unfortunately I do not know enough Nix to do it myself 😢.

Thanks in advance and do let me know if you think this might be a useful addition to the template! 🙏🏻

**Comment by rpearce** [comment]
@kutyel, This is a good idea! I will add this functionality, and I will ask you to try it once it is completed.

Thank you for trying this!

**Comment by rpearce** [comment]
@kutyel I'm sorry I still haven't gotten to this. If you figured this out for your project, please consider submitting a pull request here!

**Comment by kutyel** [comment]
Nope, although Nix works my editor setup does not work with Hakyll and HLS so I'm a bit blind while touching Haskell in the project unfortunately 😢
